src/Instagram.php

```diff
@@ -525,16 +548,14 @@ public function pagination($obj, $limit = 0)
             }
 
             $function = str_replace(self::API_URL, '', $apiCall[0]);
+            $count = ($limit) ? $limit : count($obj->data);
 
-            $auth = (strpos($apiCall[1], 'access_token') !== false);
-
-            if (isset($obj->pagination->next_max_id)) {
-                return $this->_makeCall($function, $auth, array('max_id' => $obj->pagination->next_max_id, 'count' => $limit));
+            if (isset($obj->pagination->next_max_tag_id)) {
+                return $this->_makeCall($function, array('max_tag_id' => $obj->pagination->next_max_tag_id, 'count' => $count));
             }
 
-            return $this->_makeCall($function, $auth, array('cursor' => $obj->pagination->next_cursor, 'count' => $limit));
+            return $this->_makeCall($function, array('next_max_id' => $obj->pagination->next_max_id, 'count' => $count));
```

Comment:
Pagination seems to work for me, but maybe I didn't use that endpoint yet.
Did you test it out? What is the error message you are getting back from the Twitter API?
If this is indeed an error, please make a pull request with the changed parameter name.
Thank you.